### DISCUSSION TITLE
Going back to previous design

### DISCUSSION THREAD

**Discussion by colinkiama** [nahuelwexd/vala-website#37] [General]
How would you all feel about going back to the old design?
![image](https://user-images.githubusercontent.com/16501024/167095799-ef7ecbc3-ff2d-414b-a698-c3d7456647f6.png)

**Comment by nyx-lyb3ra** [comment]
I copy and paste what I said in Discord

I discarded the old design because it seemed too simple and basic, and not as eye-catching as I had hoped. The new one fulfills the need to be eye-catching, and thus "sell" the language.

**Comment by Suzie97** [comment]
I really like the usage of colour in the new website. It  makes the website much more memorable in my opinion.

**Comment by pervoj** [comment]
I personally like both designs, but the new is in my opinion more promotional. On the old I also liked the horizontal layout. Maybe we can try to make some horizontal layout for wider screens. (like was said in #10)

**Comment by colinkiama** [comment]
Well as of today, the thin h1 headings are back xD
